### ISSUE TITLE
Trial meta-data download

### ISSUE THREAD

**Issue by wolfemd** [solgenomics/sgn#1685]
Is there a way to get all the meta-data associated with the trial? Either as variables in the trial data download itself, or a separate file. 

Downloading using the BrAPI R package is tremendously slow, but includes at least some of the meta-data (e.g. Date of Planting, Description, spacing, etc.). Downloading using the wizard and/or trial manager doesn't give the meta-data. As far as I can tell they are only available on the individual trial pages. 

Also, never posted here before. Lemme know if this belongs elsewhere.

**Comment by bellerbrock** [comment]
Related to #1606
Hey Marnin, you came to the right place. This sort of download isn't implemented yet, but was requested recently. Should be a bigger priority now that it's come up twice.

**Comment by aco46** [comment]
should be ready by next week!

**Comment by wolfemd** [comment]
Awesome, that's really helpful!

**Comment by wolfemd** [comment]
Just curious: why would you get meta-data attached with BrAPI R downloads but not from Wizard downloads?

**Comment by aco46** [comment]
@wolfemd the option for the download of meta-data associated with trials will be provided on the wizard, folder and download page.

**Comment by nickmorales** [comment]
@wolfemd which brapi call are you using?

**Comment by wolfemd** [comment]
Something like this:

```{r}
library(tidyverse); library(brapi)
db<-ba_db()$cassava base
trials_1<-db %>% 
    ba_studies_search(programDbId = "162") %>% 
    mutate(seasons=as.numeric(seasons)) %>% 
    filter(seasons >= 2015,
           seasons <= 2016) %>% 
    mutate(studyDbId=as.character(studyDbId),
           data=map(studyDbId,~ba_studies_table(db,.)))
save(trials_1,proctime,file="~/IITA_phenos_2015to2016.Rdata")
```

**Comment by aco46** [comment]
Implemented via PR #1755.
